win32/src/winapi/user32/misc.rs

```diff
@@ -24,6 +24,18 @@ pub fn CharLowerA<'a>(machine: &mut Machine, lpsz: u32) -> u32 {
     }
 }
 
+#[win32_derive::dllexport]
+pub fn CharLowerBuffA(machine: &mut Machine, lpsz: u32, cchLength: u32) -> u32 {
+    let s = unsafe {
```

Comment:
I think this is

`machine.mem().sub32_mut(lpsz, cchLength)`

(sorry for this weird memory API, I am still considering whether I want to support memory mapping)